doc/rst/users-guide/datapar/promotion.rst

```diff
@@ -128,6 +128,22 @@ values:
   :language: text
   :lines: 27,25
 
+Any non-promoted arguments to a promoted call are only promoted once. Consider
+a promoted call to ``maybeCopy()`` where the first two arguments are promoted
+and the third argument is not promoted.
```

Comment:
Nitty, but you could drop the final "promoted" without losing any clarity to avoid a little repetition.  Your call, though.